packages/apollo-collaboration-server/src/fileHandling/fileHandling.service.ts

```diff
@@ -2,7 +2,12 @@ import { existsSync } from 'fs'
 import * as fs from 'fs/promises'
 import { join } from 'path'
 
-import gff, { GFF3FeatureLine } from '@gmod/gff'
+import gff from '@gmod/gff'
+import { GFF3Sequence } from '@gmod/gff'
+import {
+  GFF3FeatureLine,
+  GFF3SequenceRegionDirective,
+} from '@gmod/gff/dist/util'
```

Comment:
These should only be imported from the top-level `@gmod/gff` package. Also, the imports can be combined.

```suggestion
import gff, { GFF3Feature, GFF3FeatureLine, GFF3Sequence } from '@gmod/gff'
```